### ISSUE TITLE
Provide an API endpoint for getting package warnings

### ISSUE THREAD

**Issue by kakaroto** [foundryvtt/foundryvtt#7913]
### User Experience

Package warnings provided through the setup page via the `getSetupData` socket command are only printed to the console log, which some users may not be familiar with or able to parse its content easily enough.
Being able to provide that same information via an API would be very helpful to allow a module to check if there are warnings about itself, or to parse those warnings and to display them in a more user friendly manner to the GM (think the difference in ease of use between the "package compatibility spreadsheet" vs the `module compatibility checker` module).
Thanks for considering.

**Comment by aaclayton** [comment]
No guarantees this makes V10 stable, but I'll add it to the list to see if we can get to it.

**Comment by kakaroto** [comment]
Yeah, I understand it's late in the cycle for this to make it to v10. So thanks regardless for considering it.
